run_tests.sh

```diff
@@ -3,8 +3,8 @@
 pip3 install .
 mv byudml tmp_byudml
 reset
-python3 primitive_jsons/generate_primitive_jsons.py
-python3 pipelines/generate_pipelines.py
+python3 submission/pipelines/generate_pipelines.py
+python3 submission/primitive_jsons/generate_primitive_jsons.py
```

Comment:
Let's switch the order back on these two lines. Generating the primitive json has validation code that should be run before we try to make a pipeline with our primitives.